torch/_inductor/codecache.py

```diff
@@ -614,22 +629,32 @@ def get_include_and_linking_paths(
         # to do to enable OMP build on darwin where PyTorch is built with IOMP
         # and we need a way to link to what PyTorch links.
         ipaths = cpp_extension.include_paths(cuda) + [sysconfig.get_path("include")]
+        if aot_mode:
+            ipaths += [os.path.join(_TORCH_PATH, "_inductor", "aot_cpp_include")]
```

Comment:
This probably also needs to be updated after the setup.py change.

Reply:
Yes, changed.